Dockerfile

```
@@ -34,11 +37,13 @@ RUN apt-get update \
         libpq-dev \
         zlib1g-dev \
     && python3 -m pip install --no-cache-dir -r ${REQ_FILE} \
-    && apt-get purge --assume-yes --auto-remove \
+    && if [ "${KEEP_DEPENDENCIES}" != "True" ]; then \
+        apt-get purge --assume-yes --auto-remove \
         gcc \
         libc6-dev \
         libpq-dev \
-        zlib1g-dev \
+        zlib1g-dev; \
+        fi \
```

```suggestion
    fi \
```
Maybe unintended, this not seems the name of a package?

---

This is indeed part of the `if; then; fi` syntax, and not a package. Dedenting it makes sense.